extensions/amp-story/1.0/bookend/amp-story-bookend.js

```diff
@@ -373,24 +369,22 @@ export class AmpStoryBookend extends AMP.BaseElement {
     if (!this.isActive_()) {
       return;
     }
-    this.vsync_.run({
-      measure: state => {
-        state.shouldBeFullBleed =
-            this.getOverflowContainer_()./*OK*/scrollTop >= FULLBLEED_THRESHOLD;
-      },
-      mutate: state => {
-        this.getShadowRoot().classList.toggle(
-            FULLBLEED_CLASSNAME, state.shouldBeFullBleed);
-      },
-    }, {});
+    let shouldBeFullBleed;
+    this.measureMutateElement(() => {
+      shouldBeFullBleed =
+          this.getOverflowContainer_()./*OK*/scrollTop >= FULLBLEED_THRESHOLD;
+    }, () => {
+      this.getShadowRoot().classList.toggle(
+          FULLBLEED_CLASSNAME, !!shouldBeFullBleed);
```

Comment:
nit: Consider initializing it to `false` so that you don't have to coerce it from `undefined` with `!!`

Reply:
Done.